_includes/v19.1/app/insecure/basic-sample.cpp

```diff
@@ -0,0 +1,41 @@
+// Build with g++ -std=c++11 basic-sample.cpp -lpq -lpqxx
```

Comment:
I think we should update this to use a named output file, e.g. `g++ -std=c++11 basic-sample.cpp -lpq -lpqxx -o basic-sample`.  That way you have a binary called `basic-sample` which is slightly nicer.  (Actually we should have a real project with a Makefile and stuff but that is future work obviously!)

Reply:
Sounds good to me! I think there's a lot of work in general to be done on the application examples to make projects a little more realistic and language-specific (like what you did for the Java example). Makefiles, for example, for C++ and C (which don't yet exist...) examples.